test/redis_test.rb

```diff
@@ -295,9 +295,9 @@ def test_circuit_breaker_on_query
   private
 
   def new_redis(options = {})
+    options[:host] = SemianConfig['toxiproxy_upstream_host'] if options[:host].nil?
```

Comment:
nit: Why this change? The benefit of the previous approach is that it didn't mutate the options hash, so callers of `new_redis` won't be surprised that it was changed :)

Reply:
`options[:host]` was always truthy when I was playing around with this, would result in `host` being empty and `Redis.new` falling back to `localhost`